boot/zephyr/boards/nrf52_minimal_footprint.conf

```diff
@@ -0,0 +1,29 @@
+# Minima MCUBoot flash footprint configuration
+# for nRF52832 SoC targets
+
+# Generated by Kconfiglib (https://github.com/ulfalizer/Kconfiglib)
+CONFIG_BOOT_SIGNATURE_TYPE_ECDSA_P256=y
+CONFIG_BOOT_SIGNATURE_KEY_FILE="root-ec-p256.pem"
+# CONFIG_BOOT_VALIDATE_SLOT0 is not set
```

Comment:
Although this might not be minimal I would strongly encourage people to have it set! :-P

Reply:
It is the minimal, not the recommend one. Will consider addition of a comment on implication.

Reply:
added comment.